### DISCUSSION TITLE
Following

### DISCUSSION THREAD

**Discussion by gbm454** [community/community#59691] [Mobile] [answered]
Quick access to people whom we are following on side panel would be helpful,
Just like in my work we have issue, projects,repo,organisation etc on the side panel
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.119.0 (102894307)
Device: iPad Air 5 (iOS 16.6)
</details>

**Comment by alcere** [comment, marked as answer]
Hey @gbm454! I have created an internal issue to evaluate and track your suggestion. Thanks for your feedback! Please keep them coming!
